### ISSUE TITLE
Health configuration reference does not reflect the new /q namespace

### ISSUE THREAD

**Issue by tlvlp** [quarkusio/quarkus#14678]
The documentation at https://quarkus.io/guides/microprofile-health#configuration-reference does not reflect the new /q/.. namespace these endpoint defaults have been moved to.

**Comment by xstefank** [comment]
@tlvlp the paths in the configuration reference seems to be correct. The `/q` is configured separately by `quarkus.http.non-application-root-path`.

**Comment by tlvlp** [comment]
@xstefank  so it seems that I have misunderstood the option. I have encountered the above problem, when I was looking to directly set the liveness and readyness endpoints as I'm porting one of our SpringBoot service to Quarkus and have to use the '/liveness' and '/readiness' endpoints of the replaced service.

From you hint and after taking a closer look to the health endpoint structure I have tried the settings below and for the non-application-root-path it seems that ' / ' is accepted to get rid of the '/q' from thepath, but when using the same approach for the health.root-path, then the root '/' becomes the '/health' (which is obviously wrong for many reasons) and the /liveness and /readiness endpoints return 'Not found' and not sure what mapping they ended up at. 
 
`
quarkus.http.non-application-root-path=/
quarkus.smallrye-health.root-path=/
quarkus.smallrye-health.liveness-path=/liveness
quarkus.smallrye-health.readiness-path=/readiness
quarkus.health.extensions.enabled=true
`
